apps/api-server/src/routes/api/generatecode.js

```diff
@@ -0,0 +1,22 @@
+const fetch = require('node-fetch');
+const createError = require('http-errors');
+const config = require('config');
+const db = require('../../db');
+const auth = require('../../middleware/sequelize-authorization-middleware');
+const pagination = require('../../middleware/pagination');
+const crypto = require('crypto')
+
+const express = require('express');
+const router = express.Router({ mergeParams: true });
+
+router.route('/')
```

Comment:
Het resultaat is nu een link naar de image server. Mag het dan of een naam krijgen waarin dat expliciet wordt, of generieker opgelost zodat het ook een link naar iets anders kan zijn? Ook dat kan in een ticket voor later natuurlijk :)